css/grid.css

```diff
@@ -24,11 +24,15 @@ header h1, header>h2{
 	align-self: center;
 }
 header nav{
+	display: grid;
+	grid-template-columns: repeat(4,1fr);
 	grid-area: navbar;
 }
+
```

Comment:
Delete empty lines like this and the line `35`. Empty lines is reserved for separations between general blocks.